ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -50,6 +54,7 @@ presubmits:
     always_run: true
     branches:
     - master
+    cluster: ci/api-build01-ci-devcluster-openshift-com:6443
```

Comment:
`pull-ci-openshift-release-master-build01-dry` not working yet.

Reply:
We decide not to migrate `applyconfig` jobs for now.

https://coreos.slack.com/archives/GB7NB0CUC/p1582923397316800?thread_ts=1582921878.312900&cid=GB7NB0CUC